src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,50 @@
+.menu {
+  box-sizing: border-box;
+  height: 100vh;
+  background-color: $menu-bg-color;
+  font-weight: 600;
+  text-transform: uppercase;
+  overflow: auto;
+
+  &__nav {
+    margin-bottom: 40px;
+  }
+
+  &__top {
+    margin-bottom: 24px;
+    position: sticky;
+    top: 0;
+    z-index: 1;
+    background-color: $menu-bg-color;
+
+    @include onTablet {
+      margin-bottom: 0;
+    }
+  }
+
+  &__phone-number {
+    display: block;
+    color: $menu-text-color;
+    text-decoration: none;
+    font-size: 16px;
+    font-weight: 600;
+    line-height: 27px;
+    margin-bottom: 16px;
```

Comment:
Remember about [css order](https://github.com/necolas/idiomatic-css#declaration-order).
Check it out in the whole code.

Reply:
done